lib/acl.js

```diff
@@ -510,7 +509,11 @@ function allow(mode, req, res, next) {
     var reqPath = res && res.locals && res.locals.path ? res.locals.path : req.path;
     var acl = reqToACL(req);
     acl.allow(mode, reqPath, function(err) {
-        next(err);
+        if (err) {
```

Comment:
Are you sure? The documentation said if anything is passed to next it will forward it to the error handler. I was not sure if a null value counts as a valid value so I made the check explicit just to make sure. But if you are sure the check is already being done by express itself I'll change it.